src/cow-react/modules/swap/hooks/useEthFlowContext.ts

```diff
@@ -30,7 +30,7 @@ export function useEthFlowContext(): EthFlowContext | null {
     kind: OrderKind.SELL,
   })
 
-  if (!baseContext || !contract || !baseProps.isEthFlow) return null
+  if (!baseContext || !contract || !baseProps.isEthFlow || baseProps.isSafeBundle) return null
```

Comment:
im confused, wasn't the ethFlow disabled already for multisigs? (which is more general case than `isSafeBundle`?

Reply:
Yeah, there are 2 flags for each, and while they are mutually exclusive, I'm still checking for both.
Would it be better to use a single variable/type/enum for this you think?

Reply:
i would think that makes sense to check `isEthFlow` in `useEthFlowContext`, right, but i think is the responsibility of whomever sets this flag, to set it false if its unsupported (like multisig wallets)